plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergMetadata.java

```diff
@@ -728,10 +728,7 @@ public void renameColumn(ConnectorSession session, ConnectorTableHandle tableHan
 
     private ConnectorTableMetadata getTableMetadata(ConnectorSession session, SchemaTableName table)
     {
-        if (metastore.getTable(new HiveIdentity(session), table.getSchemaName(), table.getTableName()).isEmpty()) {
-            throw new TableNotFoundException(table);
-        }
-
+        // getIcebergTable throws TableNotFoundException when table not found
```

Comment:
Make this a Javadoc on the method

Reply:
i can add `@throws` but the fact _how_ it gets asserted is still worth noting here, i would keep the comment